terminus-ui/src/validators/validators/isInRange/isInRange.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  ValidatorFn,
+  AbstractControl,
+  ValidationErrors,
+} from '@angular/forms';
+import { coerceNumberProperty } from '@terminus/ngx-tools/coercion';
+import { isAbstractControl } from '../../../utilities/type-coercion/is-abstract-control';
+
+
+/**
+ * Return a validator function to verify the value is within range
+ *
+ * @param minimum - The minimum value
+ * @param maximum - The minimum value
+ * @return The validator function
+ */
+export function isInRangeValidator(
+  minimum: number | AbstractControl = 0,
+  maximum: number | AbstractControl = 0):
+   ValidatorFn {
+  return (control: AbstractControl): ValidationErrors | null => {
+    // Allow optional controls by not validating empty values
+    if (!control || isNaN(control.value)) {
```

Comment:
We should use the `ngx-tools` `isNumber` function here for our check. This will allow us to work with a string number (eg `'2'`) as well as a true number (eg `2`). If it passes the check, then we can just call `Number` on it:

```
// Verify the control
if (!control) {
  return null;
}

const value: number | null = isNumber(control.value) ? Number(control.value) : null;

// Verify the value
if (value === null) {
  return null;
}
```